packaging/linux/build.spec

```diff
@@ -49,57 +49,54 @@ pip wheel git+https://github.com/cloudify-cosmo/cloudify-openstack-plugin@%{PLUG
 pip wheel git+https://github.com/cloudify-cosmo/cloudify-aws-plugin@%{PLUGINS_TAG_NAME} --wheel-dir=%{buildroot}/var/wheels/%{name} --find-links=%{buildroot}/var/wheels/%{name} &&
 pip wheel git+https://github.com/cloudify-cosmo/tosca-vcloud-plugin@%{PLUGINS_TAG_NAME} --wheel-dir=%{buildroot}/var/wheels/%{name} --find-links=%{buildroot}/var/wheels/%{name} &&
 pip wheel git+https://%{GITHUB_USERNAME}:%{GITHUB_PASSWORD}@github.com/cloudify-cosmo/cloudify-vsphere-plugin@%{PLUGINS_TAG_NAME} --wheel-dir=%{buildroot}/var/wheels/%{name} --find-links=%{buildroot}/var/wheels/%{name} &&
-pip wheel git+https://%{GITHUB_USERNAME}:%{GITHUB_PASSWORD}@github.com/cloudify-cosmo/cloudify-softlayer-plugin@%{PLUGINS_TAG_NAME} --wheel-dir=%{buildroot}/var/wheels/%{name} --find-links=%{buildroot}/var/wheels/%{name} &&
 pip wheel git+https://github.com/cloudify-cosmo/cloudify-cli@%{CORE_TAG_NAME} --wheel-dir=%{buildroot}/var/wheels/%{name} --find-links=%{buildroot}/var/wheels/%{name} &&
 
 # Make directories
 
-mkdir -p %{buildroot}/cfy/cloudify-manager-blueprints-commercial &&
-mkdir -p %{buildroot}/cfy/cloudify/types &&
-mkdir -p %{buildroot}/cfy/cloudify/scripts &&
-mkdir -p %{buildroot}/cfy/cloudify/plugins/fabric-plugin &&
-mkdir -p %{buildroot}/cfy/cloudify/plugins/script-plugin &&
-mkdir -p %{buildroot}/cfy/cloudify/plugins/diamond-plugin &&
-mkdir -p %{buildroot}/cfy/cloudify/plugins/openstack-plugin &&
-mkdir -p %{buildroot}/cfy/cloudify/plugins/aws-plugin &&
-mkdir -p %{buildroot}/cfy/cloudify/plugins/vsphere-plugin &&
-mkdir -p %{buildroot}/cfy/cloudify/plugins/softlayer-plugin &&
-mkdir -p %{buildroot}/cfy/cloudify/plugins/tosca-vcloud-plugin &&
+mkdir -p %{buildroot}/opt/cfy/cloudify-manager-blueprints-commercial &&
```

Comment:
ok. I believe this will become non relevant once we move to omnibus base
build.

On Thu, Nov 19, 2015 at 8:38 PM, Nir Cohen <notifications@github.com> wrote:

> In packaging/linux/build.spec
> <https://github.com/cloudify-cosmo/cloudify-cli/pull/238#discussion_r45381068>
> :
>
> >  pip wheel git+https://github.com/cloudify-cosmo/cloudify-cli@%{CORE_TAG_NAME} --wheel-dir=%{buildroot}/var/wheels/%{name} --find-links=%{buildroot}/var/wheels/%{name} &&
> >
> >  # Make directories
> >
> > -mkdir -p %{buildroot}/cfy/cloudify-manager-blueprints-commercial &&
> > -mkdir -p %{buildroot}/cfy/cloudify/types &&
> > -mkdir -p %{buildroot}/cfy/cloudify/scripts &&
> > -mkdir -p %{buildroot}/cfy/cloudify/plugins/fabric-plugin &&
> > -mkdir -p %{buildroot}/cfy/cloudify/plugins/script-plugin &&
> > -mkdir -p %{buildroot}/cfy/cloudify/plugins/diamond-plugin &&
> > -mkdir -p %{buildroot}/cfy/cloudify/plugins/openstack-plugin &&
> > -mkdir -p %{buildroot}/cfy/cloudify/plugins/aws-plugin &&
> > -mkdir -p %{buildroot}/cfy/cloudify/plugins/vsphere-plugin &&
> > -mkdir -p %{buildroot}/cfy/cloudify/plugins/softlayer-plugin &&
> > -mkdir -p %{buildroot}/cfy/cloudify/plugins/tosca-vcloud-plugin &&
> > +mkdir -p %{buildroot}/opt/cfy/cloudify-manager-blueprints-commercial &&
>
> I don't know why they were put there.. I just don't want to make any
> non-critical adjustments just before we release.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cloudify-cosmo/cloudify-cli/pull/238/files#r45381068>.
>